packages/amplify-category-api/src/errors/amplify-error-converter.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  $TSAny, AmplifyError, AmplifyErrorType,
+} from 'amplify-cli-core';
+
+const amplifyErrorList = [
```

Comment:
nit
```suggestion
const amplifyErrorCodes = [
```

Reply:
nit 2, consider https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set. for faster lookup.

Reply:
yeah that is most appropriate for this: https://www.tech-hour.com/javascript-performance-and-optimization